cli/wallet/wallet.go

```diff
@@ -72,10 +73,71 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:   "import-multisig",
+				Usage:  "import multisig contract",
```

Comment:
I think we need more extensive help for this one, it's not obvious how to pass multisig keys there.

Reply:
Added `UsageText`.